src/components/pill-selector-dropdown/Pill.js

```diff
@@ -3,19 +3,29 @@ import React from 'react';
 import classNames from 'classnames';
 
 type Props = {
+    isDisabled?: boolean,
     isSelected?: boolean,
     isValid?: boolean,
     onRemove: Function,
     text: string,
 };
 
-const Pill = ({ isSelected = false, isValid = true, onRemove, text }: Props) => (
-    <span className={classNames('pill', { 'is-selected': isSelected, 'is-invalid': !isValid })}>
-        <span className="pill-text">{text}</span>
-        <span aria-hidden="true" className="close-btn" onClick={onRemove}>
-            ✕
+const Pill = ({ isDisabled = false, isSelected = false, isValid = true, onRemove, text }: Props) => {
+    const removeFn = isDisabled ? undefined : onRemove;
+    return (
+        <span
+            className={classNames('pill', {
```

Comment:
Let's move this out now to a const now for breakpoints

Reply:
I'm not sure I follow what you're asking...

Reply:
const styles = classNames()
<span className={styles} >